inc/api.class.php

```diff
@@ -84,9 +85,18 @@
     *
     * @return void
     */
-   public function __construct() {
+   public function prepare() {
```

Comment:
pretty subjective, i dont like `prepare` term for the behavior.
I suggest `initApi`

Reply:
I will make this.